api/inc/vmpu_exports.h

```diff
@@ -155,6 +155,24 @@ typedef struct {
     uint32_t acl_count;
 } UVISOR_PACKED UvisorBoxConfig;
 
+typedef struct
+{
+    /* Pointer to the user context */
+    void * ctx;
+    /* Pointer to the box heap */
+    void * process_heap;
+    /* Size of the box heap */
+    uint32_t process_heap_size;
+    /* Pointer to the currently active heap.
+     * This is set to `NULL` by uVisor, signalling to the user lib, that the
+     * process heap needs to be initialized before use! */
+    void * active_heap;
+    /* Cache of the box_id */
+    int box_id;
```

Comment:
Who can access the field `.box_id`? It's probably worth spelling out what use of this cached value we intend. Otherwise it might be used in cases where it's not trusted.